src/App.tsx

```diff
@@ -1,20 +1,20 @@
 import * as React from 'react';
-import './App.css';
+import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
 
-const logo = require('./logo.svg');
+import Header from './components/Header/Header';
+import Splash from './components/Splash/Splash';
 
-class App extends React.Component<{}, {}> {
+class App extends React.PureComponent<{}, {}> {
```

Comment:
Currently nothing is changed within the `App` component directly. We can always change what type of component this is later if we do change state.